Client/Frontend/Login Management/LoginListViewController.swift

```diff
@@ -465,13 +523,23 @@ private class LoginCursorDataSource: NSObject, UITableViewDataSource {
         }
 
         let titleForSectionAtIndex = sectionTitles[section]
-        let logins = cursor?.filter { $0?.hostname.asURL?.baseDomain()?.uppercaseString.startsWith(titleForSectionAtIndex) ?? false }
+        let logins = cursor?.asArray().filter { login in
```

Comment:
If you're calling `asArray` multiple times, you might as well just use the cursor directly :)

Save the array, not the cursor.